samples/app-dev/API-Development/REST/allOf_oneOf-anyOf_construct_schema/readme.md

```diff
@@ -0,0 +1,125 @@
+![The Import app dialog](../../import-screenshots/5.png)
+
+6. You have the option to import all flows from the source app or selectively import flows.
+
+7. Click Next. If you had not selected a trigger in the previous dialog, the flows associated with that trigger are displayed. You have the option to select one or more of these flows such that the flows get imported as blank flows that are not attached to any trigger. By default, all flows are selected. Clear the check box for the flows that you do not want to import. If your flow(s) have subflows, and you select only the main flow but do not select the subflow, the main flow gets imported without the subflow. Click Next.
+
+
+## Understanding the configuration
+After importing the 'AllOf_OneOf_AnyOf_Keywords_APISpec_Service', we can see that multiple response codes have been configured in the Rest trigger. Few Response codes have Response headers as well. While adding schema in Response header and Response body we need to make sure that both are in JSON data or JSON Schema format.
+![MultipleResponseCode_configuration](./import-screenshots/MultipleResponse_serviceApp.png)
+
+ConfigureHTTPResponse activity should be used when we have configured multiple response codes in the Rest trigger. This activity is useful in mapping Response body and Response headers of a particular Response code in getting input from other activities and output to 'Return' activity.
+![ConfigureHTTPResponse_Input](./import-screenshots/ConfigureHTTPResponse_Input_Mappings.png)
+
+For the 'Invoke_AllOf_OneOf_AnyOf_Keywords_APISpec_Service' app, we have configured the activity with the API Spec of the 'AllOf_OneOf_AnyOf_Keywords_APISpec_Service'. We can upload swagger 2.0 or Open API spec 3.0 in the InvokeRestService activity to configure it. Request parameters, Request schema, Response schema and header would be auto-populated when a valid API spec file is uploaded. After uploading the spec we just need to map the input.
+App property is attached with the Invoke Rest activity URL field which can be overridden at runtime as per the URL of the service to be invoked without changing the app.
+![InvokeRest_API_spec_config](./import-screenshots/Import_InvokeRest_API_spec.png)
+
+Click on *Invoke_AllofKeywordAPISpecConfiguredFlow* click on mapper activity,navigate to input tab and expand all configurations and verify allOf properies present in schema is available in activity inputs.
+![allOf configuration](../../import-screenshots/allOfProperties.png)
+
+Click on *Invoke_OneOfKeywordAPISpecConfiguredFlow* click on mapper activity,navigate to input tab and expand all configurations.Click on oneOf schema and select the radio button corrosponding to required schema.Save the configuration and verify that in mapper input tab,properties of selected schema are appeared in activity inputs.
```

Comment:
typo in corresponding,
Save the configuration and verify that in mapper input tab properties of selected schema are present in activity inputs. Input tree closes automatically when a schema is selected, we need to open the tree again to see the newly added schema properties and mapping the values.